src/components/selection.ts

```diff
@@ -146,7 +146,7 @@ export default class SelectionUtils {
 
     sel = window.getSelection();
 
-    if (!isNaN(sel.rangeCount)) {
+    if (isNaN(sel.rangeCount)) {
```

Comment:
сюда не попадет null. Лучше тогда так 

```suggestion
    if (isNaN(parseInt(sel.rangeCount, 10))) {
```